client/task-list/tasks/payments/methods.js

```diff
@@ -297,11 +206,46 @@ export function getPaymentMethods( {
 				options.woocommerce_klarna_payments_settings.enabled === 'yes',
 			optionName: 'woocommerce_klarna_payments_settings',
 		},
+		{
+			key: 'mollie',
+			title: __( 'Mollie Payments for WooCommerce', 'woocommerce-admin' ),
+			before: <MollieLogo />,
+			plugins: [ 'mollie-payments-for-woocommerce' ],
+			isConfigured: activePlugins.includes(
+				'mollie-payments-for-woocommerce'
+			),
+			content: (
+				<>
+					{ __(
+						'Effortless payments by Mollie: Offer global and local payment methods, get onboarded in minutes, and supported in your language.',
+						'woocommerce-admin'
+					) }
+				</>
+			),
+			visible: [
+				'FR',
+				'DE',
+				'GB',
+				'AT',
+				'CH',
+				'ES',
+				'IT',
+				'PL',
+				'FI',
+				'NL',
+				'BE',
+			].includes( countryCode ),
+			container: <Mollie />,
+			isEnabled:
+				options.woocommerce_mollie_payments_settings &&
+				options.woocommerce_mollie_payments_settings.enabled === 'yes',
+			optionName: 'woocommerce_klarna_payments_settings',
```

Comment:
This should be `woocommerce_mollie_payments_settings`, this is used for the toggle

Reply:
Whoops thanks for spotting my copypasta